### DISCUSSION TITLE
brownie could not be resolved

### DISCUSSION THREAD

**Discussion by ace-brown** [smartcontractkit/full-blockchain-solidity-course-py#209] [Q&A] [answered]
Sup yall.
![Capture](https://user-images.githubusercontent.com/79329055/137581176-9a10bd30-622f-4a96-b125-a9d8bbba1bd2.JPG)
 
It works anyway, but that yellow error is really annoying for me. Any thought on how to fix that guys?

**Comment by cromewar** [comment, marked as answer]
I had the same "problem", it seems is because the python extension is incapable to recognise packages installed with `pipx` so if you want it, you could install eth-brownie using old trusty `pip` instead, but have in mind you would like to use a different virtualenv for each project.

Best regards, and hope this info help.

**Reply by ace-brown** [reply]
Thanks a bunch man.
